### DISCUSSION TITLE
cli interface for IPED-SearchApp

### DISCUSSION THREAD

**Discussion by MariasStory** [sepinf-inc/IPED#42] [Ideas]
Maybe, some functionality of the IPED-SearchApp can be done in command-line interface (cli)?
I think of something like htop or r2.
This one is useful for docker containers and remote evidence processing.
Thanks,
Tolik

**Comment by lfcnassif** [comment]
the web-api, documented some weeks ago, can be used for this:
https://github.com/sepinf-inc/IPED/wiki/Web-API

**Comment by lfcnassif** [comment]
This can also be done using a python bridge, like example script attached.

[iped-python-example.zip](https://github.com/sepinf-inc/IPED/files/4914831/iped-python-example.zip)
PS: Just work for old 3.x versions

**Comment by lfcnassif** [comment]
If those options do not work for intended use case, please reopen.
